external/linux64/lib/libcapnp-rpc.la

```diff
@@ -0,0 +1,41 @@
+# libcapnp-rpc.la - a libtool library file
+# Generated by libtool (GNU libtool) 2.4.2 Debian-2.4.2-1.11
+#
+# Please DO NOT delete this file!
+# It is necessary for linking the library.
+
+# The name that we can dlopen(3).
+dlname=''
+
+# Names of this library.
+library_names=''
+
+# The name of the static archive.
+old_library='libcapnp-rpc.a'
+
+# Linker flags that can not go in dependency_libs.
+inherited_linker_flags=' -pthread'
+
+# Libraries that this one depends upon.
+dependency_libs=' /home/mmm/mmm/devel/nupic/nupic.core/external/common/lib/libcapnp.la /home/mmm/mmm/devel/nupic/nupic.core/external/common/lib/libkj-async.la /home/mmm/mmm/devel/nupic/nupic.core/external/common/lib/libkj.la'
```

Comment:
@oxtopus this "absolute path" errs occur only in newly added files, see https://github.com/numenta/nupic/issues/2148#issuecomment-134877717 

I've only mentioned that as the Readme states "copy everything from (the compiled) lib/ bin/ to ..", but these files have not been present before, so we might as well just ignore them(?)